src/cli/log.rs

```diff
@@ -69,7 +69,7 @@ where
         (logger.compact().with_filter(env_filter).boxed(), handle)
     } else {
         // Receive log lines from Rustup only.
-        let (env_filter, handle) = reload::Layer::new(EnvFilter::new("rustup=DEBUG"));
+        let (env_filter, handle) = reload::Layer::new(EnvFilter::new("rustup=INFO"));
```

Comment:
Since `debug!()` for notifications is no longer under `if verbose`, we'll have to change this to `INFO` to prevent `debug` lines from showing up in the default state.